src/components/menus/components/ContentMenu.tsx

```diff
@@ -211,12 +212,14 @@ function ContentMenu(props: ContentMenuProps) {
                   size="icon"
                   className="richtext-w-7 richtext-h-7 richtext-cursor-grab richtext-relative richtext-z-[1]"
                   disabled={props?.disabled}
-                  onMouseUp={() => {
+                  onMouseUp={(e) => {
+                    e.preventDefault()
```

Comment:
preventDefault is redundant in this case,

but form submit is triggere by `click` event not `mouseUp`

Reply:
> preventDefault is redundant in this case,
> 
> but form submit is triggere by `click` event not `mouseUp`

@stefan-krajnik Hi, can you create a repo, I will check again, 
not sure, I tried HTML form and ant design form, I can't see this issue